src/commands/generateRunbook.ts

```diff
@@ -50,15 +50,24 @@ const prepareRunbookFile = async (workspaceFolder: vscode.WorkspaceFolder, promp
 export const generateRunbook = (secrets: vscode.SecretStorage) => vscode.window.withProgress({ location: vscode.ProgressLocation.Window }, async progress => {
     // Check docker command exists
     try {
-        const res = spawnSync("docker", ["--version"]);
+        const res = spawnSync("docker", ["version"]);
+
         if (res.error) {
             throw res.error;
         }
-    } catch (e) {
-        return vscode.window.showErrorMessage("Docker not found", { modal: true }, "Install Docker Desktop").then((value) => {
+
+        if (res.status !== 0) {
+            throw new Error(`Docker command exited with code ${res.status} and output the following error: ${res.error || res.stderr.toString()}`);
+        }
+
+    } catch (e: unknown) {
+        return vscode.window.showErrorMessage("Error starting Docker", { modal: true, detail: (e as Error).toString() }, "Install Docker Desktop", "Try again").then((value) => {
             if (value === "Install Docker Desktop") {
                 vscode.env.openExternal(vscode.Uri.parse("https://www.docker.com/products/docker-desktop"));
             }
+            else if (value === "Try again") {
```

Comment:
Would it be a good idea to recommend installing docker desktop only if the docker cli is really not present, and then have "try again after starting the docker engine" if it's just a matter of doing that.  I wonder if there's a cli tool to start the docker engine.  :)

Reply:
> wonder if there's a cli tool to start the docker engine
Somewhat platform dependent, but something to try.

Reply:
Good point.  There might be some platform dependent things here.  I think darwin and linux will turn out to be the same but we need to verify that wsl2 is the same.

I just think it's wrong to ask users to even suggest re-installing Docker Desktop when the engine is just not running.

Reply:
I committed something which works in OSX, and should work on windows but I'll be testing that soon. You can try linux?